README.md

```diff
@@ -151,6 +151,16 @@ var DropzoneDemo = React.createClass({
 React.render(<DropzoneDemo />, document.body);
 ```
 
+Upload on Safari mobile
```

Comment:
I put this here because all dropzone component I found get the same problem with safari mobile and I found this important and relevant to tell potential users that this component does work in all cases.